src/MarginTrading.Backend/Controllers/AssetPairSettingsController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System.Threading.Tasks;
+using MarginTrading.Backend.Contracts;
+using MarginTrading.Backend.Contracts.AssetPairSettings;
+using MarginTrading.Backend.Core;
+using MarginTrading.Backend.Services.AssetPairs;
+using MarginTrading.Common.Middleware;
+using MarginTrading.Common.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace MarginTrading.Backend.Controllers
+{
+    [Route("api/[controller]"), Authorize, MiddlewareFilter(typeof(RequestLoggingPipeline))]
+    public class AssetPairSettingsController : Controller, IAssetPairSettingsEditingApi
+    {
+        private readonly IConvertService _convertService;
+        private readonly IAssetPairsManager _assetPairsManager;
+
+        public AssetPairSettingsController(IConvertService convertService, IAssetPairsManager assetPairsManager)
+        {
+            _convertService = convertService;
+            _assetPairsManager = assetPairsManager;
+        }
+
```

Comment:
Maybe it is reasonable to add READ action. When using swagger you just can't see what's in the cache.

Reply:
They exist in the datareader app. It was decided not to implement them in backend to avoid the temptation of using them not for testing purposes.